Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -705,11 +734,29 @@ def securitydata_search_command(client, args):
             code42_security_data_context,
             headers=SECURITY_EVENT_HEADERS,
         )
-        security_data_context_key = "Code42.SecurityData(val.EventID && val.EventID == obj.EventID)"
-        context = {security_data_context_key: code42_security_data_context, "File": file_context}
-        return readable_outputs, context, file_events
+        code42_results = CommandResults(
+            outputs_prefix="Code42.SecurityData",
+            outputs_key_field="EventID",
+            outputs=code42_security_data_context,
+            readable_output=readable_outputs,
+            raw_response=file_events,
+        )
+        file_results = CommandResults(
+            outputs_prefix="File", outputs_key_field=None, outputs=file_context
+        )
+        return code42_results, file_results
```

Comment:
@kiran-chaudhary The way it is set up, these commands are allowed to return 1 or more CommandResults objects. This command is the only one that returns more. The reason for this is because they have different `outputs_prefix` which is important when digging through Incident Context looking at the Ouputs - and mapping outputs to different things, etc. Our integration modifies the system "File" outputs (line 744) as well as our own custom defined Code42 outputs.

Reply:
Thanks.